index.html

```diff
@@ -46,6 +46,312 @@ <h1 id="primary" class="flg">
+                <div class="card-text">
+                    <h2 class="project-title">
+                        Multi-Post Stories Gain+Glory
+                    </h2>
+                    <div class="tags1 flg">
+                        <div class="inner-tag1 flg">
+                            <h5 class="flg">Ruby on rails</h5>
+                        </div>
+                        <div class="inner-tag1 flg">
+                            <h5 class="flg">css</h5>
+                        </div>
+                        <div class="inner-tag1 flg">
+                            <h5 class="flg">JavScript</h5>
+                        </div>
+                        <div class="inner-tag1 flg">
+                            <h5 class="flg">html</h5>
+                        </div>
+                    </div>
+                    <div class="see-project">
+                        <button class="btn" type="button">
+                            <span class="btn-text flg">See Project</span>
+                        </button>
+                    </div>
+                </div>
+                <!--Image placeholder goes here-->
+                <div class="img-holder"></div>
+            </div>
+        </div>
+    </section>
+
+    <!-- About me section-->
+    <section id="about-me">
+        <!-- About me text -->
+        <div id="about-me-text" class="flg">
+            <h1 class="primary flg">About me</h1>
+            <p class="support flg">
+                I can help you build a product , feature or website Look through some of 
+                my work and experience! If you like what you see and have a 
+                project you need coded, don’t hestiate to contact me.
+            </p>
+            <button class="btn-resume" type="button">
+                <span class="btn-text flg">Get My Resume</span>
+            </button>    
+        </div>
+
+        <!--About me bacground images-->
+        <div id="background-image2"><!--Image inserted through css--></div>
+        <div id="background-image3"><!--Image inserted through css--></div>
+
+        <!--About me cards to show case skill-->    
+        <div class="about-me-card flg">
+            <div class="about-me-cards flg">
+                <div class="title-tags">
+                    <h2 class="project-title">
+                        Languages
+                    </h2>
+                    <div class="tags flg">
+                        <div class="inner-tag flg">
+                            <h5>JavaScript</h5>
+                        </div>
+                        <div class="inner-tag flg">
+                            <h5>Ruby</h5>
+                        </div>
+                        <div class="inner-tag flg">
+                            <h5>Html</h5>
+                        </div>
+                        <div class="inner-tag flg">
+                            <h5>CSS</h5>
+                        </div>
+                    </div>
```

Comment:
- [ ] Kindly wrap your categories section in the `ul` tag and use `li`  to wrap each of the categories so that your code follows best practices and semantics.